jira_agile_metrics/cli.py

```diff
@@ -17,7 +17,7 @@ def configure_argument_parser():
     """Configure an ArgumentParser that manages command line options.
     """
 
-    parser = argparse.ArgumentParser(description='Extract Agile metrics data from JIRA and produce data and charts.')
+    parser = argparse.ArgumentParser(description='Extracting Agile metrics data from JIRA and produce data and charts.')
```

Comment:
Why did you change this? The original is grammatically correct (and the change is not). Do you mind changing it back in this PR?

Reply:
Indeed, an oversight, I will fix it